frontend/cypress/integration/Call_Logs/createCallLog.spec.js

```diff
@@ -0,0 +1,45 @@
+/// <reference types="cypress" />
+
+import { loginIntercept } from '../../helpers/loginIntercept';
+import { getClientEmailProjectIntercept } from '../../helpers/projectIntercept';
+import { getCallsFromCallerIntercept } from '../../helpers/callIntercept';
+
+describe('CreateCallLogs feature e2e test', () => {
+  beforeEach(() => {
+    loginIntercept();
+  });
+
+  afterEach(() => {
+    cy.clearLocalStorage();
+  });
+
+  it('Should create a new call log', () => {
+    getClientEmailProjectIntercept();
+
+    getCallsFromCallerIntercept();
+
+    cy.intercept(
+      {
+        method: 'POST',
+        url: '/call',
+      },
+      { fixture: 'callList.json', statusCode: 201 }
+    ).as('createCallLogAPI');
```

Comment:
Can this intercept be stored as a const in the callIntercept.js along with the other intercept?

Reply:
it could be :) However, given that this is the only test that needs to intercept this call, I figured it would be clearer if it was in the file that triggered it. I tried to keep my callIntercept helper to only hold intercept that's required in more than one test to avoid code duplication.

Reply:
@jmezaber You are right. Since this intercept is not called in any other test, you can leave it here. However, if in the future this intercept is called by another test, then we need to add it in the callIntercept.js